ports/stm32/eth.c

```diff
@@ -517,6 +586,10 @@ STATIC int eth_tx_buf_get(size_t len, uint8_t **buf) {
         if (!(tx_descr->tdes3 & (1 << TX_DESCR_3_OWN_Pos))) {
             break;
         }
+        #elif defined(STM32H5)
+        if (!(tx_descr->tdes3 & (1 << TX_DESCR_3_OWN_Pos))) {
```

Comment:
It looks like this is the same code as for the H7, can it be combined?

Reply:
Done